src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -182,6 +182,21 @@ enum AppStrings {
 
 			}
 		}
+
+		enum TestTypeSelection {
+			static let title = NSLocalizedString("ExposureSubmission_TestTypeSelection_title", comment: "")
+			static let body = NSLocalizedString("ExposureSubmission_TestTypeSelection_body", comment: "")
+			static let optionSRSSelfTestTitle = NSLocalizedString("ExposureSubmission_TestTypeSelection_optionSRSSelfTest_title", comment: "")
```

Comment:
Also for the other options below

```suggestion
			static let optionSelfTestTitle = NSLocalizedString("ExposureSubmission_TestTypeSelection_optionSRSSelfTest_title", comment: "")
```

Reply:
as mentioned above.